src/main/webapp/script.js

```diff
@@ -82,12 +84,24 @@ function drawRegionsMap(censusDataArray) {
 // createDataArray takes in the data array returned by the census API
 // and reformats it into a data table for the visualization API.
 function createDataArray(censusDataArray) {
-  const vizDataArray = [];
-  censusDataArray.forEach((state) => {
-    vizDataArray.push([state[0], state[1]/1000]);
-  });
+  const vizDataArray = []
+  if (censusDataArray[0].length == 4) {
+    censusDataArray.forEach((state) => {
+      vizDataArray.push([state[0], percentToTotal(state[1], state[2])]);
```

Comment:
Just a suggestion: might be good to add both percent and number and allow user to choose which to display?

Reply:
Unfortunately the data visualization api doesn't allow for both to be shown at the same time so we'll have to add another dropdown menu later